dcm4che-net/src/main/java/org/dcm4che3/net/Device.java

```diff
@@ -1281,6 +1281,7 @@ public void reconfigure(Device from) throws IOException, GeneralSecurityExceptio
 
     protected void setDeviceAttributes(Device from) {
         setOlockHash(from.olockHash);
+        setStorageVersion(from.getStorageVersion());
```

Comment:
I couldn't used private storageVersion member here like they got away for other members because that field is in the base class.